llm4pddl/utils.py

```diff
@@ -238,6 +241,75 @@ def minify_pddl_problem(problem: str) -> str:
     return new_problem
 
 
+def embed_training_tasks(training_tasks: Sequence[Task]) -> List:
```

Comment:
there's nothing specific here to "training" tasks right? so we could call this `embed_tasks(tasks)`?